app/assets/stylesheets/custom.css.scss

```diff
@@ -0,0 +1,41 @@
+// Space out content a bit
+body {
+  padding-top: 20px;
+  padding-bottom: 20px;
+}
+
+// Custom page header
+.header {
+  padding-left: 15px;

```

Comment:
Properties should be ordered border-bottom, margin-bottom, padding-left, padding-right